.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 env:
   - PYTHON_VERSION=2.7
 before_install:
+  - redis-server --version
```

Comment:
Can you remove this line? I don't think we need it here. After that, it should be good to go.